packages/nimble-components/CONTRIBUTING.md

```diff
@@ -387,6 +387,17 @@ Test utilties located in [`/src/utilities/tests`](/packages/nimble-components/sr
 
 The jasmine unit tests utilize [`fixture.ts`](/packages/nimble-components/src/utilities/tests/fixture.ts) for component tests. The fixture utility gives tools for managing the component lifecycle. For some usage examples see [`fixture.spec.ts`](/packages/nimble-components/src/utilities/tests/fixture.spec.ts).
 
+### Disabling tests
```

Comment:
Thoughts on whether we'll want to run tests for other packages (`nimble-tokens`, `nimble-angular`, `nimble-blazor`) in all browsers too? It certainly seems like most of our browser-specific issues have come from web components differences but I don't think it's a guarantee that there won't be differences in other areas.

If we decide we want to do this, it doesn't need to happen in this PR. You could just add it to the description of #990 for us to tackle later.